test/parallel/test-child-process-execfile-maxBuffer.js

```diff
@@ -0,0 +1,94 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { execFile } = require('child_process');
+
+function checkFactory(streamName) {
+  return common.mustCall((err) => {
+    assert.strictEqual(err.message, `${streamName} maxBuffer length exceeded`);
+    assert(err instanceof RangeError);
+    assert.strictEqual(err.code, 'ERR_CHILD_PROCESS_STDIO_MAXBUFFER');
+  });
+}
+
+// default value
+{
+  execFile(
+    process.execPath,
+    ['-e', "console.log('a'.repeat(200 * 1024))"],
```

Comment:
I thought our linter didn't like double quotes for strings. Hmmm...

Reply:
I'm still debating. Which do you think is better, backquote or double quotes?
```
['-e', `console.log('a'.repeat(200 * 1024))`]
```

Reply:
Backquote, or `'console.log("a".repeat...`, either way.